content/docs/04.5-devx/03-cluster-groups/35-ingress-cluster-group.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Set Up Ingress"
+metaTitle: "Set Up Ingress for Cluster Groups"
+metaDescription: "Learn how to set up a Kubernetes Ingress for a Palette Cluster Group"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Overview
+
+Cluster Groups may have a cluster endpoint type of either Load Balancer or Ingress. The cluster endpoint type determines how Palette Virtual Clusters deployed in a Cluster Group are exposed. You specify the cluster endpoint in Cluster Group Settings.
+
+Using **Ingress** as the cluster endpoint type is a more cost effective way to access your Kubernetes workloads than using type **Load Balancer**, which requires a new cloud load balancer to be provisioned for each virtual cluster.
+
+When you enable **Ingress** as the endpoint for a Cluster Group, you must deploy an NGINX Ingress Controller add-on profile on each host cluster in the Cluster Group. The NGINX Ingress Controller provides the necessary routing functionality for external traffic to reach the Kubernetes API server of each virtual cluster, as well as any apps each virtual cluster contains.
```

Comment:
Nginx Addon profile is one thing which we will have it OOB. @ritawatson lets still go ahead with second approach where we want to let user configure Step #2 of the wiki
https://spectrocloud.atlassian.net/wiki/spaces/ENGINEERIN/pages/1810989064/How+to+setup+Ingress+on+the+Cluster+group+with+certificate

Reply:
@Sumit to look at latest Prerequisites. The example that showed enabling SSL passthrough as 'true' is removed because @TylerGillson explained this is only true for nginx and this is now implemented so it's automatic.

Reply:
@ritawatson the problem is that the system-level addon profile we've been referring to all over this doc is not available.

Reply:
It will be eventually, but for now I think @tiwarisumit3 is suggesting to reword everything :(